pkg/uninstall/uninstall.go

```diff
@@ -228,6 +238,28 @@ func Uninstall(ctx context.Context, opts *Options) error {
 		}
 	}
 
+	// Delete projects first. This will allow the acorn-controller to do all the cleanup it needs to do.
+	pterm.Info.Println("Deleting projects")
+	projectList := new(apiv1.ProjectList)
+	projects := make([]kclient.Object, 0, len(projectList.Items))
```

Comment:
nit: won't `len(projectList.Items)` always be 0 here?

Reply:
Yes, will update.